snippets/findify-product-card.liquid

```diff
@@ -1,37 +1,75 @@
 {%- if card_product and card_product != empty -%}
-  <div 
+  {% comment %} 
```

Comment:
I think we can remove this comment. If you wish to save this somewhere, maybe even drop it on a Slack channel and pin it.

One question, the <a ? navigation to product details?

Reply:
done))